Wabbajack/View Models/MainWindowVM.cs

```diff
@@ -34,6 +34,10 @@ public class MainWindowVM : ViewModel
         [Reactive]
         public ViewModel ActivePane { get; private set; }
 
+        private readonly ObservableAsPropertyHelper<Percent> _percentCompleted;
+
+        public Double PercentCompleted => (double)_percentCompleted.Value; 
```

Comment:
I would remove the cast from Percent to Double.
Despite being two lines, they should be considered one field.  What type is this field?  Double, or Percent?

You can make it double if you want, in which case you need to convert from Percent to double in the construction.  (somewhere before the ToGuiProperty call)